docfx_yaml/extension.py

```diff
@@ -916,7 +920,8 @@ def find_unique_name(package_name, entries):
         #   "google.cloud.spanner.v1.instance_v1.types"
         # it will return "instace_v1" or "params_v1" and "types".
         if name != "google" and name != "cloud" and entries[name] == 1:
-            return [name, package_name[-1]]
+            # Return only what's necessary, do not return duplicate parts twice.
+            return [name, package_name[-1]] if name != package_name[-1] else [name]
```

Comment:
Perhaps check this condition in the `if` a line above then let the default return handle it a couple lines down?

The comment is nice about why you have that particular check.

Reply:
Done.